### ISSUE TITLE
task.succeed and task.failed

### ISSUE THREAD

**Issue by ryan-linehan** [fian46/addons-btree#53]
Hi! Thank you for this awesome plugin! This is a small recommendation of renaming task.failed to task.fail. The tenses of these verbs are not consistent. The past tense of fail is failed. The past tense of succeed is succeeded.

**Comment by fian46** [comment]
Yes we can add new fail but keep failed too so it won't break people project.
